lib/ClangImporter/SwiftDeclSynthesizer.cpp

```diff
@@ -426,7 +426,8 @@ ValueDecl *SwiftDeclSynthesizer::createConstant(Identifier name,
       /*ThrowsLoc=*/SourceLoc(), /*ThrownType=*/TypeLoc(),
       params, type, dc);
   func->setStatic(isStatic);
-  func->setAccess(getOverridableAccessLevel(dc));
+  func->setAccess(isStatic ? AccessLevel::Public
+                           : getOverridableAccessLevel(dc));
```

Comment:
Hm, I think accessors match the formal access of their storage declaration, so I don't think you need to explicitly set anything here but make sure that `var` gets the expected access level set.